File: shared/src/main/java/com/emanuelvini/nuven/core/shared/player/preferences/lobby/visibility/VisibilityValue.java
package com.emanuelvini.nuven.core.shared.player.preferences.lobby.visibility;


import com.emanuelvini.nuven.core.shared.player.preferences.Preference;
import com.emanuelvini.nuven.core.shared.player.preferences.PreferenceValue;
import com.emanuelvini.nuven.core.shared.player.preferences.value.OnOffPreferenceValue;

public class VisibilityValue extends PreferenceValue<OnOffPreferenceValue> {
    public VisibilityValue(Preference preference, OnOffPreferenceValue value) {
        super(preference, value);
    }
}

File: shared/src/main/kotlin/com/emanuelvini/nuven/core/shared/player/preferences/lobby/visibility/VisibilityValue.kt
package com.emanuelvini.nuven.core.shared.player.preferences.lobby.visibility

import com.emanuelvini.nuven.core.shared.player.preferences.Preference
import com.emanuelvini.nuven.core.shared.player.preferences.PreferenceValue
import com.emanuelvini.nuven.core.shared.player.preferences.value.OnOffPreferenceValue

class VisibilityValue(preference: Preference, value: OnOffPreferenceValue) :
    PreferenceValue<OnOffPreferenceValue?>(preference, value)
